symplyphysics/laws/waves/optical_path_length_from_geometric_path_length_and_refractive_index.py

```diff
@@ -0,0 +1,38 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output, dimensionless)
+
+# Description
+## Optical path length is the length that light needs to travel through a vacuum to create the same phase difference as it would have when traveling through a given medium. 
```

Comment:
Could you elaborate on phase difference?

Reply:
About the phase difference information is only about interference